etc/env/example.env

```
@@ -2,8 +2,8 @@
 # - - -
 
 # site options
-CANDIG_MODULES=logging keycloak vault redis postgres htsget katsu query tyk opa federation candig-ingest candig-data-portal
-    #drs-server wes-server monitoring
+CANDIG_MODULES=logging keycloak vault redis postgres htsget katsu transcriptomics query tyk opa federation candig-ingest candig-data-portal
+    #minio drs-server wes-server monitoring
```

We don't need to re-include this
```suggestion
    #drs-server wes-server monitoring
```